### DISCUSSION TITLE
Support added for pkcs11-provider in Yocto Project

### DISCUSSION THREAD

**Discussion by nikomauno** [openssl-projects/pkcs11-provider#329] [General]
For your information, we submitted a new recipe which enables [Yocto Project](https://www.yoctoproject.org/) users to build and install `pkcs11-provider` into their resulting embedded Linux operating systems. 

Our contribution was since accepted and the recipe is now available in `master` branch of `meta-openembedded` repository:
https://git.openembedded.org/meta-openembedded/commit/?id=d44d4206b5bb52f102b89a7ae922c4b7f64a70db

**Comment by simo5** [comment]
Thanks, that's really cool!
